MAUI/DataForm/migration.md

```diff
@@ -1371,4 +1371,8 @@ The common properties of `Xamarin SfDataForm` list items ( [ DataFormPickerEdito
 * [MacCatalyst, iOS] - In RTL FlowDirection, the ScrollView scroll bar shows at the right.
 * [Android] - in RTL mode Radio button's content does not arrange from the right.
 * `RTL` flow direction not working for ComboBox and AutoComplete editors.
-* `IsReadOnly` not working for AutoComplete editor.
\ No newline at end of file
+* `IsReadOnly` not working for AutoComplete editor.
+* Dark theme and light theme - There is no border and focused border stroke customization for the default editors (Text, password, multiline text, picker, DatePicker and TimePicker)
+* Dark theme and light theme - [iOS] There is no border and focused border stroke customization for the default editor (Text, password, picker, DatePicker and TimePicker)
```

Comment:
* Dark and light theme - [iOS] There is no border and focused border stroke customization for the default editor (Text, password, picker, DatePicker, and TimePicker).